WoWsShipBuilder.Common/Features/ShipComparison/ShipComparison.razor

```diff
@@ -328,16 +448,28 @@
 
     private const string CategoriesUrlParameter = "ca";
 
-    private const string UpgradedModulesUrlParameter = "um";
+    private const string UpgradedModulesUrlParameter = "u";
 
     private const string HideUnfittingShipsUrlParameter = "hus";
 
     private const string SectionUrlParameter = "s";
 
+    private const string PinnedShipsUrlParameter = "p";
+
+    private const string PinnedShipsOnlyUrlParameter = "po";
+
+    private const string RangeUrlParameter = "r";
+
+    private const string HiddenColumnsUrlParameter = "hc";
+
     private Dictionary<string, bool> hiddenColumns = new();
 
+    private string hiddeColumnsUrlString = default!;
```

Comment:
is this really necessary?

Reply:
since generating that string requires iterating over the hidden columns dictionary, that parameter was made to calculate it only ones instead of multiple times